### PR TITLE
Support for anchor links on about:pages

### PR DESCRIPTION
**Pull request by diracdeltas**
This is needed to link directly to specific tabs in about:preferences,
like about:preferences#privacy in the Bravery panel (#1951).

Auditors: @bbondy

### CODE REVIEW

**Comment by bbondy** [comment]
lgtm

File: js/lib/appUrlUtil.js [hunk 1]
```diff
@@ -74,7 +74,9 @@ const aboutUrlsReverse = new Immutable.Map(module.exports.aboutUrls.reduce((obj,
  * about:blank -> http://localhost:8000/about-blank/index.html
  */
 module.exports.getTargetAboutUrl = function (input) {
-  return module.exports.aboutUrls.get(input)
+  const hash = getHash(input)
```
Comment (bridiver) [inline js/lib/appUrlUtil.js]: this appears to break tests for url fragment location updates

Comment (diracdeltas) [inline js/lib/appUrlUtil.js]: thanks, fixing

Comment (diracdeltas) [inline js/lib/appUrlUtil.js]: https://github.com/brave/browser-laptop/commit/048ab2447e373c98179988dfea810c5f73503562

**Pull request merged**